shopify-cli.gemspec

```diff
@@ -30,8 +30,9 @@ Gem::Specification.new do |spec|
     %x(git ls-files -z).split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
   spec.bindir = "bin"
-  spec.executables = spec.files.grep(%r{^bin/}) { |f| f.sub('bin/', '') }
+  # spec.executables = spec.files.grep(%r{^bin/}) { |f| f.sub('bin/', '') }
```

Comment:
This has a side effect: when some other gem tries to overwrite this file (like `shopify_api` pre 9.1.2), `gem` will check if that is OK, but by taking the `executables` away we lose that check. Is that a problem?

Would it be a valid idea to keep `executables` but then overwrite the file with the symlink?

Reply:
So, you're suggesting to leave `gem` install the wrapper in `/usr/local/bin/shopify` and we just overwrite it?  I would need to test this; not sure if the `Makefile` is run before or after that `gem` wrapper is installed.

Reply:
Yes, that would be the first idea that came to mind. I'm not sure I'm advocating that though 😅 .

My thinking here is just to somehow indicate that there IS a `shopify` binary so gem will protect us against other gems overwriting it. Is there some other way of indicating that we 're adding an executable than `executables`?